packages/asset-service/src/service/AssetServiceTestData.ts

```diff
@@ -277,7 +281,7 @@ export const mockAssets: Asset[] = [
     sendSupport: false,
     receiveSupport: false
   }
-]
+] as BaseAsset[]
```

Comment:
Ok, but the `as const` was here in order to avoir to cast all these `.tokens` access here: 
https://github.com/shapeshift/lib/pull/355/files#diff-0d5ad083ac650d73eb92d451dc27cf4e8cc54519da8394fd2016d6b2f7a170fbR7-R28

With these types, we can't anymore, so I'm forced to force this access, what's the good practice then? 😢

Reply:
`as const` is very different to `as SomeType`. `as const` says this won't change at runtime and the compiler can infer the type statically. `as SomeType` says "i know better than the compiler this is definitely `SomeType`". it needs to be use very judiciously and in general it's a code smell.

i'd rather do the `asset.tokens!` non-null assertion in a test under a specific code path, with a comment as to why, rather than asserting some test data matches a type that it may not, especially as tests change in the future.

Reply:
Ok, understood, fixed it anyway so, it's reviewable now! Thanks for the feedback again

Reply:
If you want me to comment it, I will do it later